empress/support_files/js/layouts-util.js

```diff
@@ -334,6 +373,12 @@ define(["underscore", "VectorOps", "util"], function (_, VectorOps, util) {
             if (maxWidth < xCoord[node]) {
                 maxWidth = xCoord[node];
             }
+            if (checkLengthsChange && !anyDifferent) {
+                anyDifferent = isTransformedLenDifferent(nodeLen, tree, n);
```

Comment:
```suggestion
                anyDifferent = isTransformedLenDifferent(nodeLen, tree, prepos);
```

Looks like I broke this sorry ._.  tested locally and confirmed this change should fix things with the rectangular layout

Reply:
Committing change due to discussion with @gwarmstrong